service/frontend/workflow_handler_test.go

```diff
@@ -1465,6 +1465,25 @@ func (s *workflowHandlerSuite) TestGetHistory() {
 				},
 			},
 		},
+		NextPageToken: []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
```

Comment:
Is this sequence of bytes meaningful? I think we just need any non-empty slice, right? In that case, I'd prefer if this were just `[]byte{0}` because it's much simpler.